download/main.md

```diff
@@ -3,9 +3,27 @@ title: Download InLong
 sidebar_position: 1
 ---
 
-Use the links below to download the Apache InLong Releases, the latest release is 1.1.0.
+Use the links below to download the Apache InLong Releases, the latest release is 1.2.0.
+
+## 1.2.0 release
+
+- Released: Jun 20, 2022
+- [Release Notes](release-1.2.0.md)
+
+|     Date     | Version| Comment |                                                                                                                                                                                            Downloads                                                                                                                                                                                                     |
```

Comment:
add a column to show the name of the different files.

Reply:
done